### PR TITLE
refactor: load images from CDN

### PR DESCRIPTION
**Pull request by Si-ege**
- [x] Load images from CDN

### CODE REVIEW

File: website/articles/Apache-APISIX-Incubator-Journey.md [hunk 1]
```diff
@@ -23,4 +23,4 @@ Apache APISIX 的孵化过程。
 
 关注 Apache APISIX 公众号，回复“ApacheCon”下载 PPT。
 
-<img src="../img/blog_img/APISIX-wechat.png" alt="Apache APISIX WeChat" style={{width: "200px"}} />
+<img src="https://static.apiseven.com/202108/1639468506819-7f829080-19de-4d94-b103-e1d967e0baea.pngg" alt="Apache APISIX WeChat" style={{width: "200px"}} />
```
Comment (juzhiyuan) [inline website/articles/Apache-APISIX-Incubator-Journey.md]:
```suggestion
<img src="https://static.apiseven.com/202108/1639468506819-7f829080-19de-4d94-b103-e1d967e0baea.png" alt="Apache APISIX WeChat" style={{width: "200px"}} />
```

File: website/blog/2021/08/11/interview-TuZhengsong.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ tags: [Interview]
 
 In 282 days, Zhengsong Tu (GitHub ID: tzssangglass) has grown from an open source community nerd to an Apache APISIX committer.
 
-![Apache APISIX committer tzssangglass github](/img/blog_img/2021-08-11-1.webp)
+![Apache APISIX committer tzssangglass github](https://cdn.jsdelivr.net/gh/filess/img10@main/2021/12/14/1639470441200-7a0745da-e27b-4f0b-90a9-05cc8b4d1eea.png)
```
Comment (juzhiyuan) [inline website/blog/2021/08/11/interview-TuZhengsong.md]: Could we use CDN links as other files?

File: website/blog/2021/08/11/interview-TuZhengsong.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ After choosing this issue, Zhengsong started discussing details with his communi
 
 Zhengsong was very excited and immediately followed the guide to bring up the PR. Then he accepted the comments from Code Review and further modified the code. Finally, on October 18, the PR was successfully merged into the repository.
 
-![Apache APISIX committer tzssangglass pr](/img/blog_img/2021-08-11-2.webp)
+![Apache APISIX committer tzssangglass pr](https://cdn.jsdelivr.net/gh/filess/img14@main/2021/12/14/1639470470304-1d9ac110-3ef0-4204-bf8e-cd68a26d4de3.png)
```
Comment (juzhiyuan) [inline website/blog/2021/08/11/interview-TuZhengsong.md]: ditto

File: website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ GitHub insights provides a great developer perspective, but it’s not perfect:
 
 The following graph shows the commit frequency statistics of Apache APISIX from it’s first day, which shows that Apache APISIX has maintained a very stable and continuous activities, but this metric is also a bit thin: you can’t see the data of which developers submitted these commits
 
-![Apache APISIX commits](/img/blog_img/2021-08-14-2.webp)
+![Apache APISIX commits](https://cdn.jsdelivr.net/gh/filess/img3@main/2021/12/14/1639470530551-a9dd1ed4-16d2-40f7-9b52-b658745e4e98.png)
```
Comment (juzhiyuan) [inline website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md]: ditto

File: website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md [hunk 5]
```diff
@@ -49,11 +49,11 @@ As maintainers and developers of open source projects, we also need such golden
 
 ## Contributor growth
 
-![Apache APISIX contributor growth](/img/blog_img/2021-08-14-3.webp)
+![Apache APISIX contributor growth](https://cdn.jsdelivr.net/gh/filess/img1@main/2021/12/14/1639470567442-843709f5-6048-4d44-8ab7-684d5eb09e95.png)
```
Comment (juzhiyuan) [inline website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md]: ditto

File: website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md [hunk 6]
```diff
@@ -49,11 +49,11 @@ As maintainers and developers of open source projects, we also need such golden
...
 
 ## Monthly Active Contributors
 
-![Apache APISIX Monthly Active Contributors](/img/blog_img/2021-08-14-4.webp)
+![Apache APISIX Monthly Active Contributors](https://cdn.jsdelivr.net/gh/filess/img9@main/2021/12/14/1639470234198-20a8e253-2488-450f-bd15-16e00848723c.png)
```
Comment (juzhiyuan) [inline website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md]: ditto

File: website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md [hunk 7]
```diff
@@ -65,21 +65,21 @@ The contributor growth line plot is based on the date of the first commit by eac
 
 By showing line plot for multiple repo in the same chart at the same time, we can also visually compare the development of different communities.
 
-![Apache APISIX compare the development of different communities](/img/blog_img/2021-08-14-5.webp)
+![Apache APISIX compare the development of different communities](https://cdn.jsdelivr.net/gh/filess/img10@main/2021/12/14/1639470592481-2b0b8f6f-27b7-412c-86fa-f435a105b728.png)
```
Comment (juzhiyuan) [inline website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md]: ditto

File: website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md [hunk 8]
```diff
@@ -65,21 +65,21 @@ The contributor growth line plot is based on the date of the first commit by eac
...
 
 In the graph above, we can see that the number of contributors to Apache APISIX is growing at a very fast rate, and in just two years, the number of contributors has almost caught up with or even surpassed other open source API gateway projects.
 
 The monthly contributor line plot is derived from the number of contributors committing each month.
 
 Compared to the contributor growth line plot, the monthly contributors are a better measure of how the community has grown in a short period of time.
 
-![Apache APISIX monthly contributors compare](/img/blog_img/2021-08-14-6.webp)
+![Apache APISIX monthly contributors compare](https://cdn.jsdelivr.net/gh/filess/img14@main/2021/12/14/1639470615210-03e7040a-1f46-4f83-8385-55025c91dfb0.png)
```
Comment (juzhiyuan) [inline website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md]: ditto

File: website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md [hunk 9]
```diff
@@ -65,21 +65,21 @@ The contributor growth line plot is based on the date of the first commit by eac
...
 
 For example, in the graph above, we can see that Apache APISIX has the highest and most consistent number of monthly contributors of any open source API gateway project.
 
 This explains why Apache APISIX has been able to catch up with its friends in terms of total contributors in such a short period of time since it’s first day of open source.
 
-![Apache APISIX monthly contributors](/img/blog_img/2021-08-14-7.webp)
+![Apache APISIX monthly contributors](https://cdn.jsdelivr.net/gh/filess/img9@main/2021/12/14/1639470691773-45ae3523-e0ce-45c3-9d70-e748de7fa0d2.png)
```
Comment (juzhiyuan) [inline website/blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md]: ditto

File: website/blog/2021/08/25/Using-the-Apache-APISIX-OpenID-Connect-Plugin-for-Centralized-Authentication.md [hunk 10]
```diff
@@ -71,20 +71,20 @@ An Okta account already exists.
 ### Step 1: Configure Okta
 
 1. Login to your Okta account and create an Okta application, select the OIDC login mode and the Web Application application type.
-    ![Create an Okta application](/img/blog_img/2021-08-16-4.png)
-    ![Select OIDC login mode and Web Application application type](/img/blog_img/2021-08-16-5.png) 2.
+    ![Create an Okta application](https://static.apiseven.com/202108/1639467243454-ac16645a-4a8a-426f-93a2-e840cae3c502.png)
+    ![Select OIDC login mode and Web Application application type](https://static.apiseven.com/202108/1639467299429-0ea741a7-95fd-43b5-a0c4-25a7026e62d2.png) 2.
```
Comment (juzhiyuan) [inline website/blog/2021/08/25/Using-the-Apache-APISIX-OpenID-Connect-Plugin-for-Centralized-Authentication.md:75]:
```suggestion
    ![Select OIDC login mode and Web Application application type](https://static.apiseven.com/202108/1639467299429-0ea741a7-95fd-43b5-a0c4-25a7026e62d2.png)
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2021/08/11/interview-TuZhengsong.md [hunk 11]
```diff
@@ -23,7 +23,7 @@ tags: [Interview]
 
 历时 282 天，屠正松（GitHub ID：tzssangglass）从开源社区的小白成长为了 Apache APISIX 的 committer。
 
-![Apache APISIX committer tzssangglass github](/img/blog_img/2021-08-11-1.webp)
+![Apache APISIX committer tzssangglass github](https://cdn.jsdelivr.net/gh/filess/img10@main/2021/12/14/1639470441200-7a0745da-e27b-4f0b-90a9-05cc8b4d1eea.png)
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-plugin-content-blog/2021/08/11/interview-TuZhengsong.md]: ditto

File: website/i18n/zh/docusaurus-plugin-content-blog/2021/08/11/interview-TuZhengsong.md [hunk 12]
```diff
@@ -80,7 +80,7 @@ tags: [Interview]
 
 当时正松内心非常的兴奋，马上按照那个新手指南的引导把 PR 提上去了。然后接受 Code Review 的意见，进一步修改代码。终于，在 10 月 18 号，PR 顺利地 merge 进了仓库。
 
-![Apache APISIX committer tzssangglass pr](/img/blog_img/2021-08-11-2.webp)
+![Apache APISIX committer tzssangglass pr](https://cdn.jsdelivr.net/gh/filess/img14@main/2021/12/14/1639470470304-1d9ac110-3ef0-4204-bf8e-cd68a26d4de3.png)
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-plugin-content-blog/2021/08/11/interview-TuZhengsong.md]: ditto

File: website/i18n/zh/docusaurus-plugin-content-blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md [hunk 13]
```diff
@@ -28,27 +28,27 @@ tags: [Community]
 
 - Issue 和 PR 数：GitHub 提供了 Insights 功能，如下图所示：
 
-![Apache APISIX GitHub Insights](/img/blog_img/2021-08-14-1.webp)
+![Apache APISIX GitHub Insights](https://cdn.jsdelivr.net/gh/filess/img14@main/2021/12/14/1639470503282-7e6a025d-b613-414d-aca1-3026c2b0f176.png)
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-plugin-content-blog/2021/08/14/contributors-the-golden-metric-of-openSource-projects.md]: ditto

**Review by juzhiyuan — COMMENTED** [review]
Reviewed all links and it really takes a lot of time to do this, thank you very much!

**Comment by juzhiyuan** [comment]
cc @guoqqqi

**Comment by juzhiyuan** [comment]
Also this

image

**Comment by guoqqqi** [comment]
@Si-ege ping

**Review by juzhiyuan — DISMISSED** [review]
LGTM!

**Review by guoqqqi — DISMISSED** [review]
LGTM！Thank you for your submission 😄

**Comment by juzhiyuan** [comment]
image

**Review by juzhiyuan — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Comment by lijing-21** [comment]
Hi @Si-ege ,thank you for your contribution to the Apache APISIX!

Here is the Contributor T-shirt form[1], if you're interested, kindly take a look :)

[1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Pull request merged**